.github/workflows/refresh_geodb.yml

```diff
@@ -5,6 +5,10 @@ on:
   schedule:
     - cron: "0 */4 * * *"
 
+permissions:
+  id-token: write
+  contents: write
```

Comment:
You should be able to remove the contents: write as that's just for the repo (I believe)
```suggestion
```

Reply:
true! I'll update with new PR